test/app/messagesModel.js

```diff
@@ -2,36 +2,115 @@ const assert = require('assert');
 
 require('colors');
 const should = require('should');
-const resourceRequire = require('../../app/lib/resourceRequire');
+const simple = require('simple-mock');
+
+const twClient = require('../../app/lib/twClient');
+const mailgun = require('../../app/lib/mailgun');
 
+const resourceRequire = require('../../app/lib/resourceRequire');
 const Messages = resourceRequire('models', 'Messages');
 const Communications = resourceRequire('models', 'Communications');
 const Conversations = resourceRequire('models', 'Conversations');
 
-const mock = resourceRequire('lib', 'mock');
-
-describe('Messages checks', () => {
-  // it('Should be able to create client', function(done) {
-  //   Messages.findByUserAndClient()
-  //   .then((client) => {
-  //     client.cm.should.be.exactly(2);
-  //     client.first.should.be.exactly('Joe');
-  //     done();
-  //   }).catch(done);
-  // });
-
-  it('should be able to Messages.sendOne email', (done) => {
-  	mock.enable();
-  	let communication,
-    conversation;
-  	Communications.findOneByAttribute('type', 'email')
-  	.then((resp) => {
-  		communication = resp;
-  		return Conversations.findById(1);
-  	}).then(conversation => Messages.sendOne(communication.commid, 'hi', conversation)).then(messages => Messages.where({ content: 'hi', tw_sid: '<2013FAKE82626.18666.16540@clientcomm.org>' })).then((messages) => {
-  		should.exist(messages[0]);
-  		mock.disable();
-  		done();
-  	}).catch(done);
+describe('Messages model', () => {
+  // the sid is set in seeds.js
+  it('sends an email', done => {
+    simple.mock(mailgun, 'sendEmail').resolveWith({
+      id: '<2013FAKE82626.18666.16540@clientcomm.org>',
```

Comment:
Ideally, `id` would be extracted into a variable so the DB query below would be able to refer to it directly instead of having it copied twice.